controller/src/main/java/org/jboss/as/controller/ObjectListAttributeDefinition.java

```diff
@@ -229,6 +234,19 @@ ObjectTypeAttributeDefinition getValueType() {
         return valueType;
     }
 
+
+    public List<ModelNode> unwrap(final ExpressionResolver context, final ModelNode model) throws OperationFailedException {
+        ModelNode value = resolveModelAttribute(context, model);
+        if (!model.isDefined()) {
+            return Collections.emptyList();
```

Comment:
this is a good point and a tricky at the same time.
good point become we are properly introducing the third state and as such we should have null.
tricky because of StringListAD and PropertiesAD already return emptyList/Map for undefined for same #unwrap method.
If anything we should have it unified across the board.
At this point in time, I would rather omit this method and return to it after we have eap7 out the door.
created https://issues.jboss.org/browse/WFCORE-1451 for unifying them, for now I will remove this method from this PR.